src/WCSimConstructMaterials.cc

```diff
@@ -1085,6 +1085,13 @@ void WCSimDetectorConstruction::ConstructMaterials()
    G4double EFFICIENCY_glasscath[NUM] =
      { 0.0, 0.0 };
 
+   // Coated surface properties for new photocathode physics
```

Comment:
Can you specify what these parameters mean, so it's easy for other people to follow?

Reply:
@kmtsui i.e. which equation and variable in the paper(s) these correspond to?

Reply:
These are the refractive indices of the photocathode film. Better to directly refer to`WCSimOpBoundaryProcess` to see how they are used.